### ISSUE TITLE
Don't use smart quotes in examples

### ISSUE THREAD

**Issue by derekspelledcorrectly** [appdotnet/api-spec#328]
All of the example requests have their quotes converted to fancy quotes. This makes it annoying to copy and use JSON examples, such as [here](http://developers.app.net/docs/resources/post/lifecycle/#example-json-data). The example is not actually valid JSON and can't be used without fixing all the quotes. Seems like it would be better to not do this for example code.

**Comment by berg** [comment]
Yeah, this is annoying. It's not a quick fix -- I need to go touch all of those code blocks and probably define a template for 'em. I'll try to get to this soon.
